test/mongodb.test.js

```diff
@@ -1438,6 +1439,156 @@ describe('mongodb connector', function() {
       });
     });
 
+    it('find should be able to query by location with logical operator', function(done) {
+      var coords = { lat: 1.25, lng: 20.20 };
+      var title = 'geoWithLogicalOperatorTest';
+
+      geoDb.autoupdate(function(err) {
+        var titleIndex = 0;
+
+        var createPost = function(callback) {
+          var point = new GeoPoint({
+            lat: (Math.random() * 180) - 90,
+            lng: (Math.random() * 360) - 180,
+          });
+
+          PostWithLocation.create({
+            location: point,
+            title: title + titleIndex,
+          }, callback);
+
+          titleIndex++;
+        };
+
+        async.parallel([
+          createPost.bind(null),
+          createPost.bind(null),
+          createPost.bind(null),
+          createPost.bind(null),
+        ], function(err) {
+          should.not.exist(err);
```

Comment:
Sure - no problem. I'll do that right now. I was suspicious of this pattern, but was trying to stick with the pattern established by the existing geo tests.